.github/workflows/tests.yml

```
@@ -40,3 +40,32 @@ jobs:
       - name: Run tests
         run: |
           make test
+
+  ext-test:
+    name: Go External Integration Tests
+    runs-on: ubuntu-latest
+    environment: ext-integration
+    continue-on-error: false
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)